cpansa/CPANSA-HTTP-Tiny.yml

```diff
@@ -1,4 +1,26 @@
 ---
+- id: CPANSA-HTTP-Tiny-2023-01
```

Comment:
change from 2023 to 2015, the year of the report.

Reply:
The reason for setting 2015 here was that it was the first entry I could find where it was pointed out as a problem. It wasn't discussed as a vulnerability (afaik) until 2022-2023 though, so maybe the report date should be set to 2023 instead? I'm not sure.

Reply:
I've updated the reporting_date to 2023-02-14 which was when nerdvana posted https://www.reddit.com/r/perl/comments/111tadi/psa_httptiny_disabled_ssl_verification_by_default/

Reply:
It looks like this has a CVE now, so use that for the sequence number in the `id` field. I noticed that the report below this one has that problem too.

Reply:
Would you like to use the CVE as the id instead?